base_control/Movement.ino

```diff
@@ -0,0 +1,162 @@
+Movement::Movement() {
+                    //id,d1,d2,p1,e1,e2
+  this->B_left  = Motor(1,51,50,4,3,15);
+  this->F_left  = Motor(2,37,36,5,2,22);
+  this->B_right = Motor(3,53,52,6,19,14);
+  this->F_right = Motor(4,25,24,7,18,23);
```

Comment:
No magic numbers, make them constants somewhere, maybe create a constant file. OR do something like:
Motor(/*id=*/1, /*d1=*/51, and so on. But I think it's better to do it by constants.